.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "EF Chatbot - Devcontainer",
+    "image": "mcr.microsoft.com/devcontainers/base:jammy",
+    "features": {
+       "ghcr.io/devcontainers/features/python:1": {
+           "version": "3.12.3",
+           "toolsToInstall": ""
+       },
+       "ghcr.io/devcontainers-contrib/features/poetry:2": {
+         "version": "1.8.3"
+       }
+    },
+    "customizations": {
+        "vscode": {
+            // Specify extensions to be installed in the dev container
+            "extensions": [
+                "ms-azuretools.azure-dev",
+                "ms-azuretools.vscode-azurefunctions",
+                "ms-azuretools.vscode-bicep",
+                "ms-azuretools.vscode-docker",
```

Comment:
Thank you for pushing the PR. Wondering if we still need azure functions, bicep on this repo. Please can you double check?

Reply:
We don't. I have removed all unnecessary extensions.